src/api/scanned-codes.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: default
+parent: API Reference
+title: Scanned Codes
+permalink: /api/scanned-codes
+---
+
+# Scanned Codes
+
+A scanned code is a barcode that a merchant scans. The code can be used to create a [Payment Request][].
+
+## Contents
+{:.no_toc .text-delta}
+
+* TOC
+{:toc}
+
+## Models
+
+### Scanned Code
+
+{% h4 Mandatory Fields %}
+
+|      Field       |  Type  |                                              Description                                               |
+| :--------------- | :----- | :----------------------------------------------------------------------------------------------------- |
+| code             | String | The utf8 representation of data decoded from what was scanned.                                         |
+| scannedBy        | String | The person that scanned the code. Can be `merchant`.                                                   |
```

Comment:
> I've left it as "The person that scanned the code". The only value this field can be is "merchant" so the feedback might not be relevant anymore. Let me know if you still would like me to change it.

We describe a Patron using this endpoint to start a Coke Machine purchase. In that case we'd use Patron here. No need to do this just yet.

Is the merchant a person? It's just a weird sentence to read.

Reply:
```suggestion
| scannedBy        | String | The party that scanned the code. Can be `merchant`.                                                   |
```